hibp.js

```diff
@@ -86,8 +86,8 @@ const HIBP = {
   },
 
   async getUnsafeBreachesForEmail(sha1, allBreaches) {
-    const allFoundBreaches = await this.getBreachesForEmail(sha1, allBreaches, true);
-
+    // const allFoundBreaches = await this.getBreachesForEmail(sha1, allBreaches, true);
```

Comment:
?! This change will make `getUnsafeBreachesForEmail` return `allBreaches` for any `sha1` passed in?!